libs/native/src/main/java/org/elasticsearch/nativeaccess/lib/PosixCLibrary.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.nativeaccess.lib;
+
+public non-sealed interface PosixCLibrary extends NativeLibrary {
+
+    // TODO: doc
```

Comment:
We going to address this in a follow up?

Reply:
Documented in [5abbc6f](https://github.com/elastic/elasticsearch/pull/105100/commits/5abbc6f735baa2c6bfec7e7acc9a2d05b3670a54)